java-squid/src/main/java/org/sonar/java/resolve/TypeAndReferenceSolver.java

```diff
@@ -141,6 +144,7 @@ private static void completeMetadata(JavaSymbol symbol, List<AnnotationTree> ann
         //Constant
         addConstantValue(tree, annotationInstance);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 11. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AIndentationCheck)